CI/travis/make_linux

```diff
@@ -16,8 +16,19 @@ handle_centos() {
 		/bin/bash -xe /libiio/CI/travis/inside_centos_docker.sh ${OS_VERSION} travis-ci
 }
 
+handle_bionic() {
```

Comment:
same here with `handle_ubuntu_docker` and using `$OS_VERSION` instead of `bionic`

this helps us extend for future Ubuntu versions (if we want to)